lib/storage/adapters/embedded.js

```diff
@@ -74,8 +76,11 @@ EmbeddedStorageAdapter.prototype._get = function(key, callback) {
         var getStream = exists ?
                         self._fs.createReadStream.bind(self._fs) :
                         self._fs.createWriteStream.bind(self._fs);
-
-        getStream(key, function(err, stream) {
+        if (!exists) {
+          fskey = crypto.createHash('sha1').update(key).digest('hex');
```

Comment:
For the purpose of keeping consistent with the hash function used elsewhere in storj-lib ([and the recent discovery that sha1 is dead](http://shattered.io/)), let's replace this with:

```
utils.rmd160(key, 'hex');
```